src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -731,6 +731,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\StringSplitOptions.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\SystemException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Text\ASCIIEncoding.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Text\AsciiUtility.cs" />
```

Comment:
This mix of `ASCII` and `Ascii` does not look pretty. (There are many places in the code where the two different casings are used on next to each other.)